documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,160 @@
+|---|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|----------------------|
+| 1 | Record    | Visibly acknowledge the incident and create the communication threads necessary for ongoing discussion and investigation. Understand how bad the situation is based on our severity definitions. | Stabilization pending OR Stabilized (as appropriate) | `#status-recorded`   |
+| 2 | Stabilize | <p>If there is a total or partial outage, identify service stabilization options and action them.                                                                                                | Stabilized                                           | `#status-stabilized` |
+| 3 | Resolve   | Identify outstanding improvements to the service that would mitigate the issue in the future and create tickets to implement.                                                                    | Under investigation                                  | `#status-resolved`   |
+| 4 | Review    | Identify process improvements. If the underlying cause is still unknown, identify further steps to resolve or continue investigation of the issue via 5-whys.                                    | Under investigation OR Resolved (as appropriate)     | `#status-reviewed`   |
+
+All maintainers are required to record incidents they encounter. **Crucially,
+maintainers who record incidents are not _inherently_ responsible for
+stabilizing them.** Their only required job is to record the incident. Every
+incident has a **Lead** who is the person responsible for providing updates on
+the incident and delegating tasks. The Lead is also responsible for hosting the
+retrospective after the incident is resolved.
+
+### How do we determine the lead?
+
+By default, this is the week’s MSR runner. If it is outside the MSR runner’s
+working hours, or they are unresponsive (with later explanation, ideally)
+someone else will need to be the lead. It is the recorder’s job to find this
+lead or act as this lead; although for incidents that have a severity <= 2,
+stabilizing the service may be the highest priority and take precedence over
+finding a lead. Factors which should be considered in finding a lead are
+availability and expertise. Is anyone online uniquely qualified to take this
+work on? Alternatively, has one person been handling most incidents and this
+time, someone new should take on? Consulting the threads for past incidents will
+make it easier to see the distribution of incident leadership so we can easily
+find who hasn’t done it recently.
+
+We can use our discretion in making these decisions, and if we find that the
+distribution of Lead responsibilities is inequitably distributed we can consider
+formal processes for finding them.
```

Comment:
suggestion: remove the sense that we might _not_ use our discretion. We will always use our discretion for this, it's impossible not to. Additionally, remove the unintentional implication that this process is somehow informal (could read "unintentional") or that a "formal" process is inherently more equitable.

```suggestion
This process is intentionally flexible to allow for the rapidly evolving nature of incidents and to avoid too much rigidity. If the distribution of lead responsibilities is inequitably distributed, we will develop alternative processes for assigning leads.
```